manila/templates/shares/shares.yaml

```diff
@@ -0,0 +1,9 @@
+{{- $context := . -}}
```

Comment:
This is not really needed. The inital scope of "." can be accessed in a range loop by using "$." instead.